src/nimble.nim

```diff
@@ -858,6 +860,19 @@ js   - Compile using JavaScript backend.""",
     pkgRoot
   )
 
+  # Add git vcs to the new project
+  let repo = createGitRepo(path=pkgRoot)
+  discard repo.gitCommand("init")
+
+  var fd = open(joinPath(pkgRoot, ".gitignore"), fmWrite)
+  fd.write(pkgName & "\n" & "*.out\n")
+  fd.close()
+
+  # Add a default xxx.nim.cfg file
+  fd = open(joinPath(pkgRoot, pkgSrcDir, pkgName & ".nim.cfg"), fmWrite)
+  fd.write("--d:release\n")
```

Comment:
you cant apply `release/danger` in `cfg` and it absolutely shouldn't be the default for no reason.

Reply:
@SolitudeSF why not specify a release version in the .nim.cfg file, i found that there are many nim users do not know how to input the compiler flags when compile a project, i thing it's a good beginning to create a default .nim.cfg when creating a new project.

Reply:
if they dont know about compiler flags, they for sure dont know about configuration files. and even if it was a good idea, it doesnt even work. you cant apply release flag from user configuration.

Reply:
ok, i revert it.

Reply:
@SolitudeSF It's high time we fix that.

Reply:
@SolitudeSF i think, if nim needs some compile flags, we should write them into a .cfg file, maybe it's not a good choice to write the compiler flags in the comand line. If all of us agree with the .cfg feature, i can add them before finish initializing a new project.